frontend/src/components/common/presentational/profilePageHeader/ProfilePageHeader.tsx

```diff
@@ -1,21 +1,28 @@
-import profileIcon from "@src/assets/svgs/Ellipse20Pic.svg";
 import emailIcon from "@src/assets/svgs/Email_Frame.svg";
 import phoneIcon from "@src/assets/svgs/Phone.svg";
+import { useGetUserQuery } from "@src/store/reducers/employees-api";
 import "@src/components/common/presentational/profilePageHeader/profilePageHeader.scss";
 
 function ProfilePageHeader() {
+  const { data: userData } = useGetUserQuery();
   return (
     <>
       <div className="profile-Header-main">
         <div className="icon-Container">
-          <img className="profile-pic" src={profileIcon} alt="profile pic" />
+          <img
+            className="profile-pic"
+            src={userData?.avatar}
```

Comment:
Image is not being circular shown on UI.

Reply:
That's the avatar we are getting from the backEnd. Image by container is rectangular.

Reply:
You just remove width prop from img element.

Reply:
Ohkay.